cmd/rpcdaemon/commands/eth_call.go

```diff
@@ -66,7 +66,8 @@ func (api *APIImpl) Call(ctx context.Context, args ethapi2.CallArgs, blockNrOrHa
 	if err != nil {
 		return nil, err
 	}
-	result, err := transactions.DoCall(ctx, args, tx, blockNrOrHash, block, overrides, api.GasCap, chainConfig, stateReader, api._blockReader, api.evmCallTimeout)
+	header := block.Header()
```

Comment:
can do `HeaderNoCopy` if you are sure

Reply:
Afaik the header is only read from so should be safe to take the reference.... saves some work to do it this way, will try it